nemo/collections/asr/parts/submodules/rnnt_decoding.py

```diff
@@ -208,15 +208,17 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
         self.compute_timestamps = self.cfg.get('compute_timestamps', None)
         self.word_seperator = self.cfg.get('word_seperator', ' ')
 
-        if self.durations is not None:  # this means it's a TDT model.
+        if self.durations is not None and self.durations != []:  # this means it's a TDT model.
```

Comment:
Wondering, won't it break older models whose configs already have the value set ?

Reply:
I was careful to design this change to that it is back-compatible -- in the past, duration is either a non-empty list like [0,1,2,3] or None; now it's possible to be an empty list [] which has the same behavior like None. I tested it on a model trained in the past and it still works.

Reply:
Perfect, thanks !